x-pack/plugins/maps/public/classes/layers/tiled_vector_layer/tiled_vector_layer.tsx

```diff
@@ -171,4 +197,30 @@ export class TiledVectorLayer extends VectorLayer {
     // higher resolution vector tiles cannot be displayed at lower-res
     return Math.max(this._source.getMinZoom(), super.getMinZoom());
   }
+
+  getFeatureById(
+    id: string | number | undefined,
+    meta: { mbProperties: GeoJsonProperties }
+  ): Feature | null {
+    const geometry = this._source.getFeatureGeometry(id, meta.mbProperties);
+    const properties = this._source.getFeatureProperties(id, meta.mbProperties);
+    if (geometry === null) {
```

Comment:
tbh, not sure what the right approach is here. TS-typing enforces that features have a geometry. But for .mvt-tiled data that geometry is not useful, and should not be used downstream. This just plugs in a dummy value to make the typing happy, but this value will not be used downstream.

Reply:
Maybe we need to allow for null geometry similar to how it's handled [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/geojson/index.d.ts)?